docs/content/config/security/ssl.md

```diff
@@ -158,6 +158,143 @@ Certbot provisions certificates to `/etc/letsencrypt`. Add a volume to store the
+        certbot/dns-cloudflare \
+        certonly --dns-cloudflare --dns-cloudflare-credentials /tmp/secrets/certbot/cloudflare.ini -d mail.example.com
+      ```
+
+6. Run the service to provision a certificate:
+
+    ```sh
+    docker-compose run certbot-cloudflare
+    ```
+
+7. You should see the following log output:
+
+    ```log
+    Saving debug log to /var/log/letsencrypt/letsencrypt. log | Requesting a certificate for mail.example.com
+    Waiting 10 seconds for DNS changes to propagate
+    Successfully received certificate.
+    Certificate is saved at: /etc/letsencrypt/live/mail.example.com/fullchain.pem
+    Key is saved at: /etc/letsencrypt/live/mail.example.com/privkey.pem
+    This certificate expires on YYYY-MM-DD.
+    These files will be updated when the certificate renews.
+    NEXT STEPS:
+    - The certificate will need to be renewed before it expires. Certbot can automatically renew the certificate in background, but you may need to take steps to enable that functionality. See https://certbot.org/renewal structions.
+    ```
+
+!!! tip "(Optional) Renewing certificate"
+
+    Since the above example only implements how to provision a certificate, since generated certificate will expire in 90 days, you might want to implement renew feature here.
+
+    In the following example, we will modify our created `docker-compose.yml` and create a container which will use for renewing the certificate
+
+    First, we modify our `docker-compose.yml` to add a service called `certbot-cloudflare-renew`:
+
+    ```yml
+    services:
+      certbot-cloudflare-renew:
+        image: certbot/dns-cloudflare:latest
+        command: renew --dns-cloudflare --dns-cloudflare-credentials /run/secrets/cloudflare-api-token
+        volumes:
+          - ./docker-data/certbot/certs/:/etc/letsencrtypt/
+          - ./docker-data/certbot/logs/:/var/log/letsencrypt/
+        secrets:
+          - cloudflare-api-token
+
+    If needed, run the service to renew a certificate:
+    
+    ```sh
+    docker-compose run certbot-cloudflare-renew
+    ```
+
+    You should see the following output
+    (The following log was generated with `--dry-run` options)
+
+    ```log
+     Saving debug log to /var/log/letsencrypt/letsencrypt.log
+    
+     - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -
+     Processing /etc/letsencrypt/renewal/mail.example.com.conf
+     - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -
+     Account registered.
+     Simulating renewal of an existing certificate for brtest.club
+     Waiting 10 seconds for DNS changes to propagate
+    
+     - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -
+     Congratulations, all simulated renewals succeeded:
+       /etc/letsencrypt/live/mail.example.com/fullchain.pem (success)
+     - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -
+    ``````
```

Comment:
```suggestion
    ```log
    Saving debug log to /var/log/letsencrypt/letsencrypt.log
    
    - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -
    Processing /etc/letsencrypt/renewal/mail.example.com.conf
    - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -
    Account registered.
    Simulating renewal of an existing certificate for mail.example.com
    Waiting 10 seconds for DNS changes to propagate
    
    - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -
    Congratulations, all simulated renewals succeeded:
      /etc/letsencrypt/live/mail.example.com/fullchain.pem (success)
    - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -
    ```
```